modules/logging-create-clf.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/logging-multi-clf.adoc
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: REFERENCE
+[id="logging-create-clf_{context}"]
+= Creating a log forwarder
+
+include::snippets/logging-about-clf.adoc[]
+
+If using multiple log forwarders is enabled in your cluster, you can create `ClusterLogForwarder` custom resources (CRs) in any namespace, using any name.
+If you are using a single log forwarder with a legacy system (versions older than logging 5.8), the `ClusterLogForwarder` CR must be named `instance`, and must be created in the `openshift-logging` namespace.
```

Comment:
the specific combination of openshift-logging/instance are reserved ns/name in 5.8 to support the "legacy" case.